src/components/PatchesPage/StatusSelector.tsx

```diff
@@ -56,9 +56,9 @@ const treeData = [
     key: PatchStatus.Started,
   },
   {
-    title: statusValToCopy[PatchStatus.Success],
-    value: PatchStatus.Success,
-    key: PatchStatus.Success,
+    title: statusValToCopy[PatchStatus.LegacySucceeded],
```

Comment:
Just one last question: is this also a TODO thing that will need to be updated in a year? If so, would note in the ticket so it doesn't get lost :o

Reply:
Left a comment on EVG-20032 that `PatchStatus.LegacySucceeded` should be removed! Doing so will ensure that this is updated.